modules/node_modules/@ncigdc/components/Home.js

```diff
@@ -145,21 +145,28 @@ const Home = compose(
 
       setState(state => ({ ...state, humanBodyLoading: !this.props.humanBodyData.length }));
 
+      const data = transformChartData({ hits: this.props.humanBodyData })
+        .sort((a, b) => a._key > b._key ? 1 : -1);
+
       if (this.props.humanBodyData.length) {
         setTimeout(() => {
           const root = document.getElementById('human-body-root');
           sapien.default({
             clickHandler: d => {
-              const query = {
-                filters: JSURL.stringify(makeFilter([
-                  {
-                    field: 'primary_site',
-                    value: [d._key.replace(/-/g, ' ')],
-                  }],
-                  false
-                )),
-              };
-              push({ pathname: '/projects', query });
+              const key = d._key.replace(/-/g, ' ');
+
+              if (data.find(x => x._key === key)) {
```

Comment:
I think in this case, since they are just inlined raw svgs I can hide any clickable / hoverable items with `display: none` if their related primary site isn't in the data.